yautpa-desktop/src/components/Countdown/index.scss

```diff
@@ -16,6 +16,17 @@
         height: 50%;
         width: 10%;
     }
+    button:active {
+        transform: translateY(.1rem);
+    }
+    button:hover {
+        background-color: #00D100;
+        border-color: #00D100
+    }
+    &>.counter>button:hover {
```

Comment:
same as line 19 and 22, but this time inside of `&>.counter>button` using `&:hover`. The current implementation does not affect the global buttons, but the other one just look cleaner